apps/calendar/js/views/hour_double_tap.js

```diff
@@ -40,12 +40,50 @@ HourDoubleTap.prototype = {
     this.alldaysHolder.removeEventListener('click', this._onAllDayTap);
   },
 
-  _onDayTap: function(evt) {
+  _isEventRightOnEventLink(evt) {
     var target = evt.target;
     if (!target.classList.contains('md__day')) {
+      // Could be aggressive event fuzzing registering a tap on an existing
+      // event or the "add event" UI, where the tap is really below that item.
+      // See bug 1210201.
+
+      // In this case, offsetY is the vertical space between the top of the
+      // target element and the actual tap, so if the tap is bigger than the
+      // height of the element, or smaller than 0, the tap really happened out
+      // of the element.
+      var tappedOnExistingEvent =
+        target.classList.contains('md__add-event') ||
+        target.classList.contains('md__event') &&
+        evt.offsetY >= 0 &&
```

Comment:
The lower precendence of || compared to && leads to this ending as true for just md__add-event, which results in the fuzzing before the + button registering as a press on the + when it should not. Fixed with () around the || part: `(target.classList.contains('md__add-event') || target.classList.contains('md__event')) && ...`.